NameList/NameImporter.swift

```diff
@@ -20,7 +20,7 @@ class NameImporter {
   
   func importFrom(file: NameFile) -> AnyPublisher<NameType, NameImportError> {
     guard let asset = NSDataAsset(name: file.rawValue) else {
-      return Empty().eraseToAnyPublisher()
+      return Empty(completeImmediately: true).eraseToAnyPublisher()
```

Comment:
`Empty()` creates a Publisher that never signals a new element, *but also never completes*. I think in the case of an error, you'd want it to complete.

```
// Empty()
---------->
```

```
Empty(completeImmediately: true)
----------|
```